doc/en/components/grids/_shared/column-moving.md

```diff
@@ -11,17 +11,17 @@ namespace: Infragistics.Controls
 
 The `{ComponentName}` component in {ProductName} provides the **Column Moving** feature to allow columns reordering via standard drag/drop mouse or touch gestures, or by using the Column Moving API. Column moving works both with pinned and unpinned columns and with [Multi-Column Headers](multi-column-headers.md). Moving a column into the pinned area pins the column and vice versa, moving a column outside of the pinned area unpins the column.
 
-> [!Note]
```

Comment:
why are you removing NOTE tags?

Reply:
<img width="186" alt="Screenshot 2023-04-10 at 16 59 56" src="https://user-images.githubusercontent.com/37667452/230916210-65b06cbd-4658-4b0e-aefa-dbaaf98e609c.png">
Cus look at how they appear. Its kinda ugly

Reply:
please undo removal of NOTE tags because that was a known issue 
https://github.com/IgniteUI/igniteui-docfx-template/issues/414
and we just need to update igniteui-docfx-template in our xplat-docs